descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Module KB EBSCO Module",
+  "provides": [
+    {
+      "id": "kb-ebsco-java",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/eholdings/packages"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages/{packageId}/resources"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers/{provider_id}"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/providers/{provider_id}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers/{provider_id}/packages"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/eholdings/resources"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/resources/{resourceId}"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/resources/{resourceId}"
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/eholdings/resources/{resourceId}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/titles"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/titles/{title_id}"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/eholdings/titles/{title_id}"
```

Comment:
This endpoint does not exist , its POST to collection `/eholdings/titles` instead of this one

Reply:
thanks, will change this